app/components/autocomplete/at_mention/at_mention.tsx

```diff
@@ -381,7 +382,7 @@ const AtMention = ({
 
             // No constraints? Search all groups
             if (!isTeamConstrained && !isChannelConstrained) {
-                fetchGroupsForAutocomplete(serverUrl, matchTerm || '').then((g) => {
+                searchGroupsByName(serverUrl, matchTerm || '')?.then((g) => {
```

Comment:
Under what circumstances this may be undefined?

Reply:
Whoops - it shouldn't be; a relic from previous code, have removed.